test/extended/templates/templateinstance_fail.go

```diff
@@ -0,0 +1,75 @@
+package templates
+
+import (
+	"fmt"
+	"time"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+
+	templatev1 "github.com/openshift/api/template/v1"
+	templatecontroller "github.com/openshift/origin/pkg/template/controller"
+	exutil "github.com/openshift/origin/test/extended/util"
+)
+
+var _ = g.Describe("[Conformance][templates] templateinstance creation with invalid object reports error", func() {
+	defer g.GinkgoRecover()
+
+	var (
+		cli              = exutil.NewCLI("templates", exutil.KubeConfigPath())
+		templatefixture  = exutil.FixturePath("testdata", "templates", "templateinstance_objectkinds.yaml")
+		template         *templatev1.Template
+		templateinstance *templatev1.TemplateInstance
+	)
+
+	g.Context("", func() {
+		g.BeforeEach(func() {
+			g.By("waiting for default service account")
+			err := exutil.WaitForServiceAccount(cli.KubeClient().Core().ServiceAccounts(cli.Namespace()), "default")
+			o.Expect(err).NotTo(o.HaveOccurred())
+
+			err = cli.Run("create").Args("-f", templatefixture).Execute()
+			o.Expect(err).NotTo(o.HaveOccurred())
+
+			template, err = cli.TemplateClient().TemplateV1().Templates(cli.Namespace()).Get("invalidtemplateinstance", metav1.GetOptions{})
+			o.Expect(err).NotTo(o.HaveOccurred())
+		})
+
+		g.It("should report a failure on creation", func() {
+			var err error
```

Comment:
Just a question here: Why use var err error here?

Reply:
was copied from another test, might have made more sense the way the other test was structured.

anyway i had to redo the test a bit so now that's gone.